test/test_binary_ufuncs.py

```diff
@@ -355,9 +380,9 @@ def test_div_rounding_numpy(self, device, dtype):
         a = make_tensor((4096,), device, dtype, low=low, high=high)
         b = make_tensor((4096,), device, dtype, low=low, high=high)
 
-        # Avoid integer division by zero which raises
-        if not dtype.is_floating_point:
-            b[b == 0] = 1
+        # Avoid division by zero which raises for integers and, for floats,
+        # NumPy behavior changed in 1.20
```

Comment:
What should this updated comment be given the change in behavior?

Reply:
@mruberry I've added that they changed to comply with IEEE rules for inf/nan.